{"name": "test_invalid_sign_in_get_request", "status": "passed", "steps": [{"name": "Fail sign in get request", "status": "passed", "start": 1648788928603, "stop": 1648788928779}], "start": 1648788928602, "stop": 1648788928780, "uuid": "87c95691-62a8-40db-920d-d88dc9668cf8", "historyId": "b76b91d68b162f208ab544833704df5e", "testCaseId": "c03d7c083b451bb970c52430f0e4a590", "fullName": "tests.api_testcases.login_test.login_test.TestUserLogin#test_invalid_sign_in_get_request", "labels": [{"name": "parentSuite", "value": "tests.api_testcases.login_test"}, {"name": "suite", "value": "login_test"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "pop-os"}, {"name": "thread", "value": "80820-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api_testcases.login_test.login_test"}]}